sufia-models/app/services/sufia/versioning_service.rb

```diff
@@ -0,0 +1,27 @@
+module Sufia
+  class VersioningService
+    # Make a version and record the version committer
+    # @param [ActiveFedora::File] content
+    # @param [User] user
+    def self.create(content, user=nil)
+      content.create_version
+      record_committer(content, user) if user
+    end
+
+    # @param [ActiveFedora::File] file
+    def self.latest_version_of(file)
+      file.versions.last
```

Comment:
file.content.versions only works when file is a GenericFile. file.versions is when file is an ActiveFedora::File